overlord/snapstate/snapstate.go

```diff
@@ -781,6 +781,8 @@ func doUpdate(st *state.State, names []string, updates []*snap.Info, params func
 		}
 
 		snapsup := &SnapSetup{
+			Base:         update.Base,
+			Prereq:       defaultContentPlugProviders(st, update),
```

Comment:
I like the fact how it extends existing funcionality with just this simple change :+1:

Reply:
Very nice how the integration bits fall into place.